thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -155,10 +155,11 @@ class CDistributorBase : implements IHashDistributor, implements IExceptionHandl
                     }
                     else
                     {
-                        prev.set(row);
-                        rows.enqueue(row.getClear());
+                        rows.enqueue(prev.getClear());
+                        prev.setown(row.getClear());
```

Comment:
@shamser - I'm guessing the reason you didn't see the crash is because you're running on a 1-slave Thor (?), which doesn't involve this send dedup code. Any non-best test involving >1 slave would have hit this crash.